flask/blueprints.py

```diff
@@ -105,6 +106,19 @@ def __init__(self, name, import_name, static_folder=None,
         if url_defaults is None:
             url_defaults = {}
         self.url_values_defaults = url_defaults
+        if cli_group_name is None:
+            cli_group_name = name
+        self.cli_group_name = cli_group_name
+        if cli_group_desc is None:
+            cli_group_desc = 'Commands provided by {0}.'.format(self.name)
```

Comment:
I initially had a longer description (*"Commands provided by the ... blueprint"*), but click ended up truncating it...